src/yunohost/app.py

```diff
@@ -806,9 +806,9 @@ def app_makedefault(auth, app, domain=None):
     try:
         with open('/etc/ssowat/conf.json.persistent') as json_conf:
             ssowat_conf = json.loads(str(json_conf.read()))
-    except ValueError:
+    except ValueError as e:
         raise MoulinetteError(errno.EINVAL,
```

Comment:
Because bad conf is probably a user generated conf, for a good reason, and we souldn't override it. I'd rather raise an error for the user to carefully watch and fix his custom configuration.